### PR TITLE
CI fix

### PR DESCRIPTION
**Pull request by Cre-eD**
Jenkinsfile was standardized.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
+def dockerTags = ['master': 'latest', 'develop': 'dev']
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
def dockerTags = ['master': 'latest', 'develop': 'develop']
```

File: Jenkinsfile [hunk 2]
```diff
@@ -10,28 +11,31 @@ pipeline {
         }
     }
     stages {
-        stage('Build') {
```
Comment (BulatSaif) [inline Jenkinsfile]: не очень понимаю зачем поменял `Build` с `Test`?

**Review by BulatSaif — CHANGES_REQUESTED** [review]
This is repository do not have the sora gradle plugin

**Comment by BulatSaif** [comment]
can you check with developers that this repository should push docker image?

**Comment by Cre-eD** [comment]
This repository doesn't need changes.

**Pull request closed without merging**